app/apps/planner/migrations/0040_daysettings_districts.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.13 on 2022-07-11 08:13
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("planner", "0039_rename_zaken_team_name_teamsettings_zaken_team_id"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="daysettings",
+            name="districts",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.PositiveSmallIntegerField(),
+                blank=True,
+                null=True,
+                size=None,
+            ),
+        ),
+    ]
```

Comment:
Ik zie twee modellen met een districts field in dit PR. Hoort dit niet een FK relatie te zijn naar generieke districten of iets dergelijks?

Reply:
Scherp, en klopt ook. Het is alleen juist ook de bedoeling om ze los van elkaar te hebben. ItinerarySettings heeft ook een relatie met DaySettings. DaySettings moeten aangepast moeten kunnen worden zonder dat ItinerarySettings mee veranderd, maar ItinerarySettings wil wel een referentie naar DaySettings en Team Settings, voor generiek instellingen. Ook zou het kunnen dat bepaalde waardes uit DaySettings gebruikt zullen worden in de toekomst, dit komt naar voren als mensen een looplijst willen generen met instellingen uit DaySettings, bv als er toch machtigingen gelopen moeten worden, dan kun je de originele DaySettings aanpassen en dan kunnen gebruikers met Voeg zaak toe, met deze aangepaste instelling nieuwe zaken ophalen.  Wat zou kunnen, is dat ItinerarySettings erft van DaySettings, om dubbele velden te voorkomen.

Reply:
Lijkt me goed dan zo